src/main/fulcro/client/data_fetch.cljc

```diff
@@ -79,6 +80,8 @@
      :without              without
      :post-mutation        post-mutation
      :post-mutation-params post-mutation-params
+     :initialize           (when (and initialize SubqueryClass server-property-or-ident (prim/has-initial-app-state? SubqueryClass))
+                             {server-property-or-ident (prim/get-initial-state SubqueryClass (if (map? initialize) initialize {}))})
```

Comment:
@wilkerlucio This is where we could decide to make the map just literal, instead of parameters to `get-initial-state`. Thoughts?

Reply:
Sounds right, so we can have the custom initialization if needed. 👍

Reply:
Hmm, now I thought again, maybe is a good idea to use the map literally instead of throwing on `get-initial-state`. The reason is the later is more flexible, if you want to re-use parts of the `initial-state` you can call it yourself. On the current situation, the user has no change to override the thing if he wants to, leading to the initial problem back again, what you think?.

Reply:
That was my thought. The way it is...it's less to type at the `load` call site, but it is inherently less flexible, and since it might be a complete sub-graph might be somewhat difficult to reason about. I can go either way, but the literal map seems better than using it as params to `get-initial-state`